README.md

```diff
@@ -161,15 +157,18 @@ See [reporting bugs][bug_guide] for details about reporting any issues.
 
 Operator Lifecycle Manager is under Apache 2.0 license. See the [LICENSE][license_file] file for details.
 
-[architecture]: /doc/design/architecture.md
-[philosophy]: /doc/design/philosophy.md
```

Comment:
We probably still want to link to the philosophy doc, wherever it was being pointed from.

Reply:
It looks like we're removing the [installation guide] here as well?